metadrive/utils/scene_export_utils/utils.py

```diff
@@ -36,8 +36,8 @@ def get_type_from_class(obj_class):
 def convert_recorded_scenario_exported(record_episode, scenario_log_interval=0.1):
     result = dict()
     result["id"] = "{}-{}".format(record_episode["map_data"]["map_type"], record_episode["scenario_index"])
-    result["dynamic_map_states"] = [[{}]]  # old data has no traffic light info
-    result["version"] = EDITION
+    result["dynamic_map_states"] = {}  # old data has no traffic light info
```

Comment:
Have you finished the ```extract_dynamic_object()```? So that it has the same structure as traffic?

Reply:
WIP